app/src/main/java/com/stonks/api/Constants.kt

```diff
@@ -4,8 +4,9 @@ class Constants {
     companion object {
         const val CURRENCY_API_BASE_URL = "https://api.exchangeratesapi.io/"
 
-        const val STOCK_API_BASE_URL = "https://www.alphavantage.co/"
+        const val STOCK_API_BASE_URL = "https://www.alphavantage.co/query/"
```

Comment:
No need to add "query" endpoint. All requests are anyway sent there via `@GET("endpoint")`, we've discussed it already and found it useless